Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.8.14 (unreleased)
 
 Language Features:
+* General: Extended the language to add the event selector field.
```

Comment:
All the items in this section extend the language, so it does not have to be explicitly mentioned.